tests/conftest.py

```diff
@@ -3371,3 +3372,42 @@ def restore_golden_config_db(duthost):
 def gnmi_connection(request, setup_connection):
     connection = setup_connection
     yield connection
+
```

Comment:
Suggest moving this function to [duthost_utils.py](https://github.com/sonic-net/sonic-mgmt/blob/master/tests/common/fixtures/duthost_utils.py)

Reply:
ACK